packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -205,6 +206,19 @@ const App = (props) => {
 
     return (
         <Box className="sf-app" {...styles.container}>
+            <Helmet>
+                <script async src="https://www.googletagmanager.com/gtag/js?id=G-2DSPGYVQRG" />
```

Comment:
This is installing the Google Analytics tag.

I wanted to do it from within the overrides, but couldn't. So I did it in the base template here instead.